frontend/src/metabase/query_builder/components/view/View.styled.tsx

```diff
@@ -14,7 +14,7 @@ export const QueryBuilderViewRoot = styled.div`
   display: flex;
   flex-direction: column;
   background-color: ${color("bg-white")};
-  height: calc(100vh - 49px);
+  height: calc(100vh - 61px);
```

Comment:
Can we extract this `61px` to a constant? So it's something like `calc(100% - ${SOME_ELEMENT_HEIGHT})`